modules/juce_graphics/native/juce_RenderingHelpers.h

```diff
@@ -553,6 +553,46 @@ namespace GradientPixelIterators
 
         JUCE_DECLARE_NON_COPYABLE (TransformedRadial)
     };
+
+    //==============================================================================
+    /** Iterates the colour of pixels in a conical gradient */
+    struct Conical
+    {
+        Conical (const ColourGradient& gradient, const AffineTransform&,
+                 const PixelARGB* colours, int numColours)
+            : lookupTable (colours),
+              numEntries (numColours),
+              centre (gradient.point1),
+              offsetAngle (gradient.point1.getAngleToPoint(gradient.point2))

```

Comment:
I see some lacking space before `(` as per the JUCE coding standards in this new struct.